test/e2e/parallel/space_test.go

```diff
@@ -47,9 +48,11 @@ func TestCreateSpace(t *testing.T) {
 
 	t.Run("failed to create space - unknown target member cluster", func(t *testing.T) {
 		// given & when
-		s, _, _ := CreateSpace(t, awaitilities, WithTierName("appstudio"), func(space *toolchainv1alpha1.Space) {
-			space.Spec.TargetCluster = "unknown"
-		})
+		s, _, _ := CreateSpace(t, awaitilities,
+			WithCreationTimestamp(time.Now().Add(time.Second*60)), // we set a time in the future, so that spacecleanup_controller doesn't delete the space before we actually create the spacebinding
```

Comment:
As far as I remember there is already a delay in the space garbage collector which waits for 60 seconds (?) before deleting spaces without any space bindings... If I'm not mistaken.

Reply:
I had to increase the deletionThreshold in the spacecleanup_controller see https://github.com/codeready-toolchain/host-operator/pull/776 , but not sure it's the same delay you are referring to.

Reply:
Yes, it's the same delay.